src/Evolution/Systems/GeneralizedHarmonic/GaugeSourceFunctions/RegisterDerived.cpp

```diff
@@ -0,0 +1,21 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Evolution/Systems/GeneralizedHarmonic/GaugeSourceFunctions/Factory.hpp"
+#include "Parallel/RegisterDerivedClassesWithCharm.hpp"
+
+namespace GeneralizedHarmonic::gauges {
+namespace {
+template <size_t Dim>
+void impl() {
+  Parallel::register_classes_with_charm(tmpl::list<>{});
+}
+}  // namespace
+
+void register_derived_with_charm() {
+  impl<1>();
+  impl<2>();
+  impl<3>();
```

Comment:
Not sure I understand the purpose of these...are these just creating "gauges" that are empty?

Reply:
Plan is to have dim-dependent gauges (e.g. have Analytic also register the analytic conditions that can be imposed)